test/e2e/framework/framework.go

```diff
@@ -95,7 +107,10 @@ func NewFramework(baseName string, client clientset.Interface) *Framework {
 		BaseName:  baseName,
 		ClientSet: client,
 	}
-
+	// Dev/CI only configuration. The seclist for CCM load-balancer routes.
+	f.CCMSecListID = os.Getenv("CCM_SECLIST_ID")
```

Comment:
Both these values will default to an empty string if the env var is not set. Would that cause logical problems when passed to framework.CountSecListTunnelSinglePortRules ?

Reply:
A good thing to query, but, hopefully it is ok. I hope it is the case that if these seclists values are not set then the [test calculations](https://github.com/oracle/oci-cloud-controller-manager/blob/1e79b88e7f330a0348e6aa46cd84cf621259c9cf/test/e2e/framework/seclist_util.go#L65) will collapse into 0 - so they will reported a s valid. I tested it manually and it seemed to work ok. We could make it explicit, but, it would make the tests more verbose to read.

Reply:
Done.